types/ckeditor__ckeditor5-utils/src/fastdiff.d.ts

```diff
@@ -1,85 +1,117 @@
-import { DeleteChange, InsertChange } from "./difftochanges";
+interface DeleteChange {
```

Comment:
This is another example of why exporting the `DeleteChange` and `InsertChange` interfaces is cleaner as it reduces duplication and simplifies maintenance. The types themselves have no runtime effects, so there's no reason not to export/import them.